### PR TITLE
Login

### PR DESCRIPTION
**Pull request by joacomiguens**
## Summary

- Create Login page.

- Manage redirections between detail-book page, main page and login.

- Create mixins for form-button and light-blue border.

## Screenshot

login

## Trello Card

You can find it [here](https://trello.com/c/ZJ0BgZGR/32-login)

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="login">
+        <img src="./assets/LogoWolox.png" alt="Wolox logo" class="login__logo" />
+        <div class="login__item">
```
Comment (ebedoyaalzate) [inline login.html]: This div is unnecessary.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          <label for="email" class="login__item__label">Email</label>
+          <input type="text" id="email" name="email" class="login__item__input" />
+        </div>
+        <div class="login__item">
```
Comment (ebedoyaalzate) [inline login.html]: Same here

File: scss/_login.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
+.container {
+  justify-content: center;
+}
+.login {
+  @include light-blue-border;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (ebedoyaalzate) [inline scss/_login.scss]: Order properties.

File: scss/_login.scss [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  justify-content: space-between;
+  max-width: 400px;
+  min-height: 550px;
```
Comment (ebedoyaalzate) [inline scss/_login.scss]: Better use a max-heght and height to set this.

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: login.html (hunk 1 above)
Comment (joacomiguens) [reply login.html]: Done 😄

File: login.html (hunk 2 above)
Comment (joacomiguens) [reply login.html]: Done 😄

File: scss/_login.scss (hunk 3 above)
Comment (joacomiguens) [reply scss/_login.scss]: Done 😄

File: scss/_login.scss (hunk 4 above)
Comment (joacomiguens) [reply scss/_login.scss]: Done 😄

**Review by ebedoyaalzate — APPROVED** [review]

File: scss/_login.scss [hunk 5]
```diff
@@ -0,0 +1,50 @@
+.container {
+  justify-content: center;
```
Comment (mcallegari10) [inline scss/_login.scss]: Without the `display: flex` I think this doesn't work 🤔

File: login.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="login">
```
Comment (mcallegari10) [inline login.html]: This should be a `form`

File: login.html [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+        <img src="./assets/LogoWolox.png" alt="Wolox logo" class="login__logo" />
+        <label for="email" class="login__label">Email</label>
+        <input type="text" id="email" name="email" class="login__input" />
```
Comment (mcallegari10) [inline login.html]: We usually split the fields in `div`s of their own. This helps us to create components, both JS and SASS.

File: scss/_login.scss [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+  flex-direction: column;
+  justify-content: flex-start;
+  max-width: 400px;
+  padding: 30px;
+  width: 100%;
+
+  &__logo {
+    margin-bottom: 40px;
+    max-width: 350px;
+    width: inherit;
```
Comment (mcallegari10) [inline scss/_login.scss]: Why not `100%`?

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: scss/_login.scss (hunk 5 above)
Comment (joacomiguens) [reply scss/_login.scss]: True 🥵

File: login.html (hunk 6 above)
Comment (joacomiguens) [reply login.html]: Done 😄

File: login.html [hunk 9]
```diff
@@ -9,15 +9,19 @@
...
+      <form class="login">
         <img src="./assets/LogoWolox.png" alt="Wolox logo" class="login__logo" />
-        <label for="email" class="login__label">Email</label>
-        <input type="text" id="email" name="email" class="login__input" />
-        <label for="pwd" class="login__label">Password</label>
-        <input type="password" id="pwd" name="pwd" class="login__input" />
-        <a href="index.html" class="login__button fill">Login</a>
+        <div class="login__group">
+          <label for="email" class="login__group__label">Email</label>
+          <input type="text" id="email" name="email" class="login__group__input" />
```
Comment (mcallegari10) [inline login.html]:
```suggestion
          <input type="email" id="email" name="email" class="login__group__input" />
```

**Review by mcallegari10 — APPROVED** [review]
Just that comment 👏

**Pull request merged**